testing/trino-product-tests-launcher/src/main/java/io/trino/tests/product/launcher/env/common/Standard.java

```diff
@@ -52,6 +52,10 @@
 import static org.testcontainers.containers.wait.strategy.Wait.forLogMessage;
 import static org.testcontainers.utility.MountableFile.forHostPath;
 
+// Use io.trino.tests.product.launcher.env.common.StandardMultinode instead
```

Comment:
nit This could be javadoc `@deprecated`

Reply:
Well... I was expecting you to say something opposite. So I avoided javadoc ;)